docker-compose/edge-compose.yml

```diff
@@ -26,17 +30,27 @@ services:
     command: >-
       --pop_id              ${POP_ID:-mypop}
       --server_id           ${SERVER_ID:-myserver}
-      --core_host           ${DIST_OR_CORE_HOSTS:-dist}
+      --core_host           ${DIST_HOSTS:-dist}
```

Comment:
It's just an unfortunate config parameter name `--core_host` that makes me hesitate to rename that variable. I don't have strong feelings however, so let's keep your revision.